resources/js/processes/scripts/components/ScriptEditor.vue

```diff
@@ -106,6 +106,7 @@
               <b-card-body class="overflow-hidden p-0">
                 <b-list-group class="w-100 h-100 overflow-auto">
                   <cornea-tab
```

Comment:
Can you replace this with `<ai-tab` in a similar way to this PR please?
https://github.com/ProcessMaker/processmaker/pull/5477/files